dashboard/app/models/levels/level.rb

```diff
@@ -615,6 +615,13 @@ def add_starter_asset(friendly_name, uuid_name)
     save
   end
 
+  # Remove a starter asset by its key (friendly_name) from the level's properties.
+  def remove_starter_asset(friendly_name)
+    return unless starter_assets
+    starter_assets.delete(friendly_name)
+    save
```

Comment:
`save!` would be better here, to raise an exception (rather than return a client error) if validations fail, since that seems like it would indicate a programming error or other server problem that we'd want to see in honeybadger. 

if the object is not found (whether or not starter_assets) exist, then returning false here and returning `not_found` to the client would more closely match REST conventions. (caveat: I'm basing that on one stackoverflow article: https://stackoverflow.com/questions/6439416/deleting-a-resource-using-http-delete)